en/docs/visual-programming/sequence-diagram-view.md

```diff
@@ -1 +1,83 @@
-# Sequence diagram view
\ No newline at end of file
+# Sequence diagram view
+With Ballerina you can view your code in graphical mode and develop your program logic through it. This
+allows you to look at your program logic holistically and gives you the ability to comprehend the program at 
+a glance. This visual representation has its roots in sequence diagrams and can capture how the logic of
+the program flows, how concurrent execution flow works and what remote endpoints are involved and how they interact 
+with the different objects in the program.
+
+## Switching to graphical mode
+There are two methods of switching to the graphical mode using the vs-code extension
+
+1. By executing the `Ballerina: Show Diagram` command from the command palette.
+    ![command screenshot](./img/show-diagram-command-palette.png)
+2. By using the `Show Diagram Button` in the editor group.
+    ![button screenshot](./img/editor-group-button.png)
+
+## Graphical Representation of code
+
+### Functions
+Graphical representations of functions are based on sequence diagrams, each function has a lifeline with a start and 
+end, and this lifeline represents the flow of the function logic. 
+![button screenshot](./img/function-sequence-diagram.png)
+
+### Concurrent Execution
+Concurrent executions are supported in Ballerina through workers. Each worker has a lifeline that runs parallel to 
+the lifeline of the main function body, the diagram also captures asynchronous messaging between workers.
+
+The following example shows a sample of messages passing between workers and waiting in the main function until the 
```

Comment:
`sample of messaging passing..` or `list of messages passing..`?
@praneesha

Reply:
Changed the description as 
`The following example shows how asynchronous messaging is represented in the diagram.`
in ee4df6e